Rakefile

```diff
@@ -14,6 +14,7 @@ task :clean do
 end
 
 require 'rake/testtask'
+ENV['EDITOR'] = ''
```

Comment:
Keep in mind that this sets `EDITOR` globally for the Rakefile, because it's executed at the top level. This is probably not what is intended.